src/orquestra/sdk/schema/workflow_run.py

```diff
@@ -72,3 +72,42 @@ class WorkflowRun(WorkflowRunMinimal):
 
     task_runs: t.List[TaskRun]
     status: RunStatus
+
+
+class WorkflowRunSummary(WorkflowRunOnlyID):
+    """
+    A summary overview of a workflow run
+    """
+
+    status: RunStatus
+    owner: t.Optional[str]
+    total_tasks: int
+    completed_tasks: int
```

Comment:
**nitpick**: we should pay attention to task defs vs task invocations vs task runs. It's okay to skip it at the presentation layer to make the UI more concise, but our schemas should be explicit.
```suggestion
    total_task_runs: int
    completed_task_runs: int
```